specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,330 @@
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/securitydomain/download": {
+      "post": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "operationId": "HSMSecurityDomain_Download",
+        "description": "Retrieves Security domain from HSM enclave",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "Certificates",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateSet"
+            },
+            "description": "Security domain download operation requires customer to provide three certificates containing public key in JWK format."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Security domain download response",
+            "schema": {
+              "$ref": "#/definitions/SecurityDomainObject"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Retrieve Security domain": {
+            "$ref": "./examples/securitydomaindownloadpost-example.json"
+          }
+        }
+      }
+    },
+    "/securitydomain/transferkey": {
+      "get": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "operationId": "HSMSecurityDomain_TransferKey",
+        "description": "Retrieve security domain transfer key",
+        "responses": {
+          "200": {
+            "description": "Security domain transfer key operation",
+            "schema": {
+              "$ref": "#/definitions/TransferKey"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Retrieve security domain transfer key": {
+            "$ref": "./examples/securitydomaintransferkey-example.json"
+          }
+        }
+      }
+    },
+    "/securitydomain/upload": {
+      "post": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "operationId": "HSMSecurityDomain_Upload",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
```

Comment:
This parameter implies that the initial call is using both Location and Azure-AsyncOperation headers, and opposite to the LRO spec we want to avoid doing a final Location call, and stop the polling at the latest Azure-AsyncOperation call. 
Just want to check this is intended behavior.